From: majestrate <jeff@oxen.io>
Date: Wed, 18 May 2022 12:16:46 -0400
Subject: use ::/0

makes us map all ranges not just ipv4 when exit mode is on by default.
---
 llarp/rpc/rpc_server.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/llarp/rpc/rpc_server.cpp b/llarp/rpc/rpc_server.cpp
index 5bb14e8..5545358 100644
--- a/llarp/rpc/rpc_server.cpp
+++ b/llarp/rpc/rpc_server.cpp
@@ -432,7 +432,7 @@ namespace llarp::rpc
                 const auto range_itr = obj.find("range");
                 if (range_itr == obj.end())
                 {
-                  range.FromString("0.0.0.0/0");
+                  range.FromString("::/0");
                 }
                 else if (not range.FromString(range_itr->get<std::string>()))
                 {
